src/main/resources/application.properties

```diff
@@ -149,9 +149,12 @@ cache.ecl.enabled=true
 #         These form part of the exported filename.
 #     {FieldNames} is a comma separated list of the field names after the 'referencedComponentId' field.
 #
+#   New refset types are per default on the MAIN branch but can be configured to any already existing branch.  
+#
 #   During import files are matched using pattern ".*_{FieldNames}Refset_{Name}.*".
 #   For more information see the 'Release File Specification' section of the Technical Implementation Guide http://snomed.org/tig
 # ----------------------------------------
+#refsets.branch=MAIN
```

Comment:
`refset-types.initial-branch` sounds good. Although the `initial-` part could be confusing to people who don't know refset type can be rebased. Perhaps I should add a quick note that it can be and this is only for initialization?

Reply:
Yes, more notes is good 😄